Projects and homepages 4593

Anahisv23 opened the pull request:
Fixes #4593 

### What changes did you make?
Additions to CSS files
   - Added the form#search-bar css, line 13 in _search-bar.scss file. Added position sticky to keep search bar pinned
   - Added the .projects-and-filters css, line 40 in _dropdown_filters.scss. Added overflow-y to allow independent scrolling of projects 
   
Modifications to existing CSS files

 - Modified the .filter-toolbar css, line 32 in _dropdown_filters.scss. Added overflow-y to allow independent scrolling of filters list 
-  Modified the .filtersDiv css, line 46 in _dropdown_filters.scss. Added position sticky to keep filters section pinned 
 
Note: 
- Modifying the _dropdown_filters.scss and _search-bar.scss files applies css changes to both the projects page and home pages user interface.

### Why did you make the changes (we will use this info to test)?
  - To improve the usability of the filters, search and project display so that it is easy for people to navigate the project list.
 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>

![image](https://github.com/hackforla/website/assets/74325966/5490f57b-c6cc-49de-a6e2-8e3e7152af3e)

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
![image](https://github.com/hackforla/website/assets/74325966/4f63ea3b-4cb2-46ed-a95a-3d360a178b08)


</details>

Anahisv23 committed:
modified styling for filter, searchbar, and projects list to improve usability of browsing projects on home page and projects page

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593

Thinking-Panda commented:
Availability: weekdays
Review ETA: 5/1/24 EOD

Thinking-Panda reviewed (dismissed):
@Anahisv23 Thank you for working on this issue. Merge branch is setup correctly. Code changes show expected output. Website works well on my local machine. Well done. Approved!

irais-valenzuela commented:
Review ETA: 5/7/24
Availability: 3PM 5/7/24

irais-valenzuela reviewed (dismissed):
Hey @Anahisv23, great work on the issue! Your branches are correct, there is a linked issue, and the correct css styling has been added to achieve the goals of the issue. Overall, very awesome job!

jphamtv reviewed (changes requested):
Hi @Anahisv23,

Great progress on this issue. The filter section and projects scroll independently, and the search bar remains pinned to the top of the projects list.

However, during testing, I noticed some behavior that needs to be addressed:

1. The content height appears to have been reduced, limiting the viewable area. The content height should fill the viewable area. The filter/project section should scroll to the top of the page and become sticky when it reaches the top, just below the navigation bar, to increase the screen real estate for viewing projects.
2. The project list should scroll with the rest of the page, similar to the REI example provided in the issue, where the list shares the same vertical scroll as the page. This will also avoid competing vertical scrolls on horizontally shorter browser windows.
3. The filter section should not shrink horizontally or have horizontal scroll.
4. On screen width 768px and below, when opening filters, there is a gap between filters and action buttons, allowing the projects list to be visible and scrollable underneath.

Please make the necessary adjustments to address these issues. Test the changes on both the Projects page and the Home page.

Anahisv23 commented:
Hey @jphamtv thanks for the feedback! I'll look into making these adjustments this week. My availability for this week is from 11am - 5pm.

Anahisv23 commented:
Update: I worked on addressing some of the feedback you @jphamtv mentioned but I had a very busy week with interviews so I'm looking of getting this done next week!

Anahisv23 committed:
addressed concerns on this issue

Anahisv23 committed:
Update _dropdown_filters.scss line 19

Anahisv23 committed:
Update _main.scss by removing height styling

Anahisv23 commented:
Hi @jphamtv, I went ahead and made changes to my approach to address the concerns you had. Let me know what you think!

What my new changes address
1. Content section pins just below navigation 
2. Removed scrolling from .project-and-filters so projects lists scrolls with whole page
3. Fixed width on .filter-toolbar and added overflow-x to remove horizontal scroll
4. Fixed gap in between filters and action buttons by adding max height property to .filter-toolbar.show-filters on mobile media query

Thinking-Panda commented:
Thank you @Anahisv23 for making the requested changes. Following are the behavior that I noticed.

1. In mobile viewport, the size of the filter-toolbar has changed.

   Before Change:
![MobileFilterBeforePR6747](https://github.com/hackforla/website/assets/122488603/39ed9784-73d4-4bad-b0e9-a4160a05c5fc)
   After Change:
![MobileFilterAfter-PR6747](https://github.com/hackforla/website/assets/122488603/8736e077-00dd-404e-94cc-86516fe0599e)

3. In mobile viewport, the filter and the search bar are pinned to the top and causing an overlap.
   
![Large mobile PR 6747](https://github.com/hackforla/website/assets/122488603/3a654961-0edf-475b-a7e7-be0c0b41c34d)


4. In desktop viewport, the project list scrolls when the pointer is still in the filter-toolbar.

![Recording-PR6747-ezgif com-video-to-gif-converter](https://github.com/hackforla/website/assets/122488603/f83f416d-67b9-43fe-bbaa-ba46255d999c)

@jphamtv Please share your thoughts on these observations.

jphamtv reviewed (changes requested):
Hi @Anahisv23,

Great work on the updates, the following are working well:
- The project section is perfectly implemented for the desktop.
- The filter and project sections scroll to the top of the page, increasing the viewable area.
- The filter section doesn't have horizontal scroll.
- No competing scrolls, resulting in much better functionality.

In addition to the feedback from @Thinking-Panda, please address the following:

- Revert the filter section's horizontal shrinking on desktop view to match the current website's behavior, where only the project cards shrink instead of the filter section.
- Ensure the filter's content height fills the viewable area on desktop view (maybe replace 750px with 100vh and see if that works?).
- On screen width 768px and below, when opening filters, eliminate the gap between the filter and action buttons to prevent projects from being visible underneath.

Thanks for your efforts on this. Reach out if you have any questions.

Anahisv23 commented:
Hi @Thinking-Panda and @jphamtv thanks for the feedback and visual feedback on this issue. I'll look into this tomorrow or next week. I'm currently interviewing for a company so I have been a bit swamped with work.

Anahisv23 commented:
Hi @Thinking-Panda and @jphamtv. I wanted to provide an update on the issue and what I've managed to do. 

Have completed
- Fixed the pinning problem with filter toolbar and search bar when viewport is below 678 on mobile 
- Eliminated gap between filter and action buttons when viewport is below 678 on mobile 
- Fixed filter horizontal shrinking so it looks like hack for la website when viewport is above 678 on desktop
- Ensured the filter's content height fills the viewable area when viewport is above 678 on desktop

Still need to work on 
- Fixing mobile toolbar size 
-  Fixing the problem where the project list scrolls when the pointer is still in the filter-toolbar section

Anahisv23 committed:
Refactored css based on additional feedback on filter and project list functionality

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593

This merge integrates the latest updates from the 'gh-pages' branch into the 'projects-and-homepages-4593' branch.

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593
This merge integrates the latest updates from the gh-pages branch into the projects-and-homepages-4593 branch

Anahisv23 committed:
Merge branch 'projects-and-homepages-4593' of github.com:Anahisv23/website into projects-and-homepages-4593
Pulling changes from my remote branch to my local branch

Anahisv23 commented:
Hi @Thinking-Panda and @jphamtv. Hope y'all are well! I've managed to incorporate changes based on the feedback y'all provided. Thanks for the feedback btw the website looks so much nicer haha. Below I'll include details of what I did to address the feedback given. Let me know if you have any more feedback or questions. 

**Based on @Thinking-Panda feedback** 
On mobile the filter toolbar and search bar no longer pin
- On the media query for mobile and tablet I changed .filter-toolbar position to position relative to prevent it from taking sticky positioning styles for desktop .filter-toolbar css styling 
- I added a media query to form#search-bar  and changed position to position static to prevent it from taking sticky positioning styles for desktop form#search-bar css styling 

On mobile I fixed sizing of filter toolbar size
-  On media query for mobile and tablet I adjusted .filter-toolbar width, top, bottom, left, and right properties to prevent it from inheriting the filter-toolbar desktop styling  

On desktop I prevented scrolling through projects list when we reach the end of the filters tool bar scroll section when the pointer is still on filter-toolbar 
- By adding an event listener on current-projects.js 


**Based on @Jpham feedback** 
On desktop I ensure the filter's content height fills the viewable area 
- By editing height to be 95% of the viewport so its fills the full viewable area 

On desktop I fixed filter horizontal shrinking so it looks like hack for la website where only projects shrink 
- By adding flex-shrink 0 to prevent filter toolbar section from shrinking horizontally 

On mobile I eliminated the gap between filter and action buttons 
- By removing the max height property so that the max height property doesn’t act like a constraint to the height specified

jphamtv commented:
Hi @Anahisv23, you've made excellent progress on this! You're almost there. You handled all the previous items, but a couple more issues came up with the Filters section.

1. On shorter screens, when the filter area is not in the sticky position up top, it should be scrollable within the viewable area. In the image below, the Status category is not reachable. The scrolling works fine when the filters are sticky and pinned to the top, but it should also be scrollable in the viewable area on screens with shorter vertical height.

<img width="800" alt="Screenshot 2024-06-06 at 16 44 20" src="https://github.com/hackforla/website/assets/4952258/83932a93-a43b-43da-8bbc-a4bcd2216ce8">
<br><br>
2. The scroll functionality doesn't work consistently in the Languages/Technologies/Tools section when you click "View more". It works intermittently, or not at all.
<br><br>
<img width="800" alt="Screenshot 2024-06-06 at 16 44 31" src="https://github.com/hackforla/website/assets/4952258/3238a89f-4b2a-47c9-be9f-a856123136a2">
<br><br>
Let me know if you have any questions.

jphamtv reviewed (changes requested):
Hi @Anahisv23, you've made excellent progress on this! You're almost there. You handled all the previous items, but a couple more issues came up with the Filters section.

1. On shorter screens, when the filter area is not in the sticky position up top, it should be scrollable within the viewable area. In the image below, the Status category is not reachable. The scrolling works fine when the filters are sticky and pinned to the top, but it should also be scrollable in the viewable area on screens with shorter vertical height.

<img width="800" alt="Screenshot 2024-06-06 at 16 44 20" src="https://github.com/hackforla/website/assets/4952258/83932a93-a43b-43da-8bbc-a4bcd2216ce8">
<br><br>
2. The scroll functionality doesn't work consistently in the Languages/Technologies/Tools section when you click "View more". It works intermittently, or not at all.
<br><br>
<img width="800" alt="Screenshot 2024-06-06 at 16 44 31" src="https://github.com/hackforla/website/assets/4952258/3238a89f-4b2a-47c9-be9f-a856123136a2">
<br><br>
Let me know if you have any questions.

Thinking-Panda reviewed (changes requested):
@Anahisv23 Thank you for persistently working on this issue. A couple of things that I observed are:
1. Based on @jphamtv observations, I notice that the scroll in the Languages/Technologies/Tools section freezes when the window is resized
2. On viewports <768px, I believe that the search bar has to be pinned to the top as per the issue.
3. I still observe that the "projects-inner" div scrolls when the pointer is still in the filter-toolbar section.

Anahisv23 commented:
Hey @jphamtv and @Thinking-Panda thanks for the feedback! 

@jphamtv I've managed to fix the first issue you mentioned. Regarding the second issue do you mind doing a screen recording of the Languages/Technologies/Tools scrolling bug. I'm not seeing the scrolling bug on my end at all in mobile or desktop and I want to get an understanding on what it looks like on your end. 

@Thinking-Panda I believe the issue states that the search bar should be pinned when the viewport widths are above 767px in which case it is pinned. Regarding the third bullet point do you mind doing a screen recording of the project list scrolling that occurs when the pointer is on the filters-toolbar section. I want to get an understanding on what it looks like on your end. 

Thank you!

Thinking-Panda commented:
@Anahisv23 , my understanding is that the the filter and project are pinned to each other and centered for viewports above 767px (Like, there is no gap/space in between the filter and the projects). And for mobile viewport, you follow the REI website as a reference and pin the search bar. @jphamtv- Please share your perspective on this.

Here is a recording of the scrolling.
![Recording-PR67471-ezgif com-optimize](https://github.com/hackforla/website/assets/122488603/ddd71cd1-b6f7-47e0-b670-40a97134a225)

Anahisv23 commented:
Hi @Thinking-Panda thanks for sharing the recording about the scrolling. I'll try to see what I can do about that. I'm curious if @jphamtv if you are experiencing the scrolling issue on your end as well? Regarding the search bar pinning on mobile, I can definitely fix that. Regarding the gap between the filter and projects could you provide a screenshot of the gap you are talking about. I want to make sure I'm understanding. Thank you!

Anahisv23 committed:
pinned search bar on mobile

jphamtv commented:
>@jphamtv I've managed to fix the first issue you mentioned. Regarding the second issue do you mind doing a screen recording of the Languages/Technologies/Tools scrolling bug. I'm not seeing the scrolling bug on my end at all in mobile or desktop and I want to get an understanding on what it looks like on your end.

@Anahisv23 - Here's a video of what I'm experiencing on my end. The same issue is on mobile.

https://github.com/hackforla/website/assets/4952258/8f0a9c80-0f34-4f23-a106-47637ea50abd

>@Anahisv23 , my understanding is that the the filter and project are pinned to each other and centered for viewports above 767px (Like, there is no gap/space in between the filter and the projects). And for mobile viewport, you follow the REI website as a reference and pin the search bar. @jphamtv - Please share your perspective on this.

@Anahisv23 , @Thinking-Panda - The issue specifically mentions viewport widths above 767px but not below, which leads me to believe that mobile viewports may be out of scope. However, it would be best to clarify this with those involved in ER #4530 who identified the problem.

Anahisv23 commented:
Hello :) Thanks for the feedback and for the screen recordings! Here’s an update 

@jpham feedback 
- To fix the status filter not being scrollable on shorter vertical screens
    - I added padding to the top and bottom of filtersDiv on desktop
    - Adjusted the filtersDiv css rules on mobile since it was inheriting the css rules for desktop

- Still need to look into the languages/technologies/tools scrolling inconsistencies
    - Any suggestions/resources on how to approach this issue? I’m thinking I have to modify .dropdown.show-all on line 126 on *dropdown*filters.scss file.

@thinking-panda feedback 
- Will reach out to those involved in the ER #4530 whether we need to pin the search bar on viewport width below 767px. Thanks for bringing this up!
- Removed gap in between projects and filter toolbar
- Regarding the scrolling of projects inner while mouse is on filter-toolbar. I’m blocked on how to fix this issue. Do you have any suggestions/resources on how to approach this?

Thinking-Panda commented:
@Anahisv23 Thank you for your patience on this issue. I am quite held up with another assignment. I will check the scrolling concern over the weekend and get back to you.

Anahisv23 commented:
> @Anahisv23 Thank you for your patience on this issue. I am quite held up with another assignment. I will check the scrolling concern over the weekend and get back to you.

Thank you and no worries @Thinking-Panda! @jphamtv I wanted to provide an update so I reached out to the people who created the https://github.com/hackforla/website/issues/4530 and it seems we will be addressing mobile styling separately meaning for now I will not pin the search bar on mobile.

Anahisv23 committed:
updated styling on search bar to prevent it from pinning on mobile

jphamtv commented:
>- Still need to look into the languages/technologies/tools scrolling inconsistencies
    - Any suggestions/resources on how to approach this issue? I’m thinking I have to modify .dropdown.show-all on line 126 on *dropdown*filters.scss file.

Yeah, I think you're correct. The inconsistent scroll issue is probably due to it being nested under .filter-toolbar. I don't have a specific suggestion on how to approach this, but I see the overflow properties in both divs are the same, and you have the heights defined, so it may need JavaScript to fix the inconsistency. Looking forward to @thinking-panda's insights on a solution.

An alternative solution could be to remove the scrolling from the `.dropdown.show-all` div altogether and, instead of keeping the options inside a limited viewable area, show all the options when "View all" is clicked. You could replace the "View all" link with "View less" and have the category collapse back to show the default state of displaying 8 options. This is similar to how the referenced REI website behaves and would create a better user experience. I'm not sure if this increases the scope of the issue, and would probably need to be approved before moving forward with this.

Thinking-Panda commented:
@Anahisv23 , @jphamtv 
- While reviewing the website on my local now, I did not observe this behavior on the home page. 
- Regarding the projects page, I noticed the behavior only on my laptop. 
- When i moved the browser screen to my monitor I did not observe this behavior.
- When I resize the window on the Developer tools, I did not observe the behavior.
The behavior is not consistent on my side. With that said, If you do not observe the behavior on your end, we can ignore my observation and approve the PR. Let me know.

Anahisv23 committed:
updated filtersDiv, removed gap in between projects list and filters, and changed height of filter-toolbar

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593
This merge is needed to merge the recent changes from gh-pages branch into the projects-and-homepages-4593

Anahisv23 commented:
Hi @Thinking-Panda and @jpham,

Thank you both for your feedback! I agree with @Thinking-Panda; I'm not seeing any scrolling inconsistencies on my end either. The main functionality of the ticket has been implemented I believe, and it seems to be working correctly in my environment.

@jpham, I appreciate your alternative solution suggestion regarding the "View all" functionality. While I haven't implemented it yet, it might be a good option to consider if we continue to encounter issues.

I'll push my latest changes so you both can review and see if everything looks good on your end. Let me know if there are any additional adjustments needed :) 

Thanks again!

New changes I’m introducing 
- Removed the css rule that added margin to .projects-and-filter class to remove the gap in between filter and projects 
- Added padding to filterDiv bottom on desktop so that filters option status are scrollable within the viewable area 
- Changed filtersDiv on mobile media query so we don’t apply padding bottom from filtersDiv desktop styling 
- Added the calc method to the height of filter-toolbar to ensure that the toolbar fits within the available vertical space without overlapping other elements

Thinking-Panda commented:
@Anahisv23 The project section does not scroll any more when the mouse is in the filter section ( even in the screen size I observed the issue earlier)

Thinking-Panda reviewed (changes requested):
@Anahisv23 Changes to `food-oasis.md` is not within the scope of this issue. Please remove that file from the commit. Thanks.

Anahisv23 commented:
Hi @Thinking-Panda I'm not sure why that is evening showing up here. My bad. I'm not sure how to remove the file from my commit history. Do you have any git command suggestions I could use? I did some research and it seems like git rm path/to/file is a git command I could use but I want to see if you have any suggestions? Thanking you!

Anahisv23 commented:
> @Anahisv23 The project section does not scroll any more when the mouse is in the filter section ( even in the screen size I observed the issue earlier)

Awesome! Glad we were able to tackle this issue!

jphamtv commented:
@Anahisv23 - I'm still experiencing the scrolling issue with the Languages/Technologies/Tools category.

I've made sure to clear my cache and cookies, refreshed/rebuilt Docker, and tested on Chrome, Safari, and Firefox browsers with the same results.

If you and @Thinking-Panda are not encountering issues with the scrolling behavior in the Languages/Technologies/Tools category, the problem may be on my end. In that case, someone else may need to review and approve for merging.

Could you both confirm if you're able to scroll properly in this category on your end? Specifically, can you scroll up and down through all the options in the category after expanding with the View All link/button? Please test this at various viewable screen heights as well.

jphamtv reviewed (commented):
@Anahisv23 - I'm still experiencing the scrolling issue with the Languages/Technologies/Tools category.

I've made sure to clear my cache and cookies, refreshed/rebuilt Docker, and tested on Chrome, Safari, and Firefox browsers with the same results.

If you and @Thinking-Panda are not encountering issues with the scrolling behavior in the Languages/Technologies/Tools category, the problem may be on my end. In that case, someone else may need to review and approve for merging.

Could you both confirm if you're able to scroll properly in this category on your end? Specifically, can you scroll up and down through all the options in the category after expanding with the View All link/button? Please test this at various viewable screen heights as well.

Thinking-Panda commented:
> > * Still need to look into the languages/technologies/tools scrolling inconsistencies
> >   - Any suggestions/resources on how to approach this issue? I’m thinking I have to modify .dropdown.show-all on line 126 on _dropdown_filters.scss file.
> 
> @Anahisv23 - Yeah, I think you're correct. The inconsistent scroll issue is probably due to it being nested under .filter-toolbar. I don't have a specific suggestion on how to approach this, but I see the overflow properties in both divs are the same, and you have the heights defined, so it may need JavaScript to fix the inconsistency. Looking forward to @Thinking-Panda's insights on a solution.
> 
> An alternative solution could be to remove the scrolling from the `.dropdown.show-all` div altogether and, instead of keeping the options inside a limited viewable area, show all the options when "View all" is clicked. You could replace the "View all" link with "View less" and have the category collapse back to show the default state of displaying 8 options. This is similar to how the referenced REI website behaves and would create a better user experience. I'm not sure if this increases the scope of the issue, and would probably need to be approved before moving forward with this.

@jphamtv The scroll is inconsistent in the Languages/Tools/Technologies section on my end as well. I did not mention about it as you had suggested a different solution for it. The inconsistency still occurs.

Thinking-Panda commented:
> Hi @Thinking-Panda I'm not sure why that is evening showing up here. My bad. I'm not sure how to remove the file from my commit history. Do you have any git command suggestions I could use? I did some research and it seems like git rm path/to/file is a git command I could use but I want to see if you have any suggestions? Thanking you!

@Anahisv23 Please go ahead and try that.

Anahisv23 commented:
@Thinking-Panda Gotcha, I'll go ahead and try that git command then and push again. Regarding the scrolling inconsistencies on the languages/technologies/tools section. I can try to implement one of the approaches @jphamtv suggested and push when I'm done. I have been very swamped this week with back to back interviews so I'll work on this a bit today and next week. Thanks for y'all help and feedback!

Anahisv23 committed:
removed food-oasis.md file from commit

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593
Merge branch gh-pages into projects-and-homepages-4593

Anahisv23 commented:
@Thinking-Panda I tried that git command and that removed the file from my commit. Can you confirm I did this correctly on your end?

Anahisv23 committed:
modified max height on dropdown show all css rule to fix scroll inconsistencies

jphamtv reviewed (commented):
Hi @Anahisv23,
I'm still experiencing inconsistent scrolling behavior on my end. It's improved somewhat, but there are still some issues. 

For example:
When I expand the category, I can scroll down the options. But if I want to scroll back up halfway through the options, I can't. I can eventually scroll back up, but only after scrolling to the bottom of the filter section.

I'm curious why you're not experiencing this behavior on your end. Are you able to replicate this issue? If not, it might be helpful to have another team member test this functionality to see if it's an isolated problem.

Anahisv23 commented:
Hey @jphamtv thanks for bringing this up! I actually am seeing this issue on my end now as well. For some strange reason I wasn't seeing them before. I will try to retrace my steps and see where the inconsistencies started happening. I'll see what I can do about it today and this week.

mSharifHub commented on _sass/elements/_dropdown_filters.scss:
```diff
@@ -12,8 +12,9 @@
+
 .page-contain.projects-inner {
   margin: 0px 0px;
 }
```
.page-contain.projects-inner {
  margin: 0; /* Use shorthand for consistency */
}

Anahisv23 replied:
Hi @mSharifHub thanks for the feedback! I think this change isn't within the scope of the issue so I'll go ahead and leave this as is. I did however go ahead and add the overflow-y: auto where you suggested. Thanks!

mSharifHub commented on _sass/elements/_dropdown_filters.scss:
```diff
@@ -31,11 +32,17 @@ a.filter-item {
-  height: 100%;
+  height: calc(100vh - 64px);
+  flex-shrink: 0;
+  overflow-y: scroll;
```
overflow-y: auto;   Use auto to show scrollbar only when necessary

mSharifHub reviewed (changes requested):
Hello, Just some minor changes but overall you in the right track. Good Job

Anahisv23 committed:
addressing scrolling inconsistencies

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593
merging gh-pages branch into projects-and-homepages

Anahisv23 commented:
Hi @jphamtv I've been working on this and I think I finally solved the scrolling inconsistencies! Let me know if you are experiencing them on your end.

Thinking-Panda reviewed (changes requested):
@Anahisv23 The way the file is removed from the commit might remove the file from the repo once the PR is merged. I have asked for help from merge team on further steps to be taken. Thank you for your patience and understanding.

Anahisv23 commented:
Hi @Thinking-Panda thanks for asking the merge team about this! My bad I didn't mean to remove it entirely. Would you like me to pull the latest changes from gh-pages, create a new food-oasis.md file on my branch, and just copy and past the code from food-oasis.md on gh-pages onto my branch?

LRenDO commented:
> Hi @Thinking-Panda thanks for asking the merge team about this! My bad I didn't mean to remove it entirely. Would you like me to pull the latest changes from gh-pages, create a new food-oasis.md file on my branch, and just copy and past the code from food-oasis.md on gh-pages onto my branch?

Hi @Anahisv23! Yes, that is likely the least complicated way to resolve the issue. You can go ahead and do that.

Anahisv23 commented:
Hi @LRenDO! I'll go ahead and try that today see if that fixes this issue.

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593
merge gh-pages into projects-and-homepages-4593

Anahisv23 committed:
created a new food-oasis.md file

jphamtv reviewed (dismissed):
>Hi @jphamtv I've been working on this and I think I finally solved the scrolling inconsistencies! Let me know if you are experiencing them on your end.

@Anahisv23 - The nested scrolling issue is now resolved - no more conflicts on my end. Excellent work figuring out the scrolling problems and implementing this feature. Thanks for all your effort addressing the feedback.

Anahisv23 commented:
Hey @jphamtv awesome! I'm glad the scrolling issues are resolved. Thank you for your feedback and help throughout this ticket. I appreciate y'all! Thank you :) I believe now we just need @Thinking-Panda and @mSharifHub to review this and approve it to get this merged.

Thinking-Panda commented:
@Anahisv23 As noticed earlier, the project list scrolls when the pointer is still in the filter section on my local machine. If other reviewers do not notice that behavior, I can go ahead and approve this PR. I will wait for the other reviewer's comments. Thank you and appreciate your patience.

Anahisv23 committed:
added overscroll-behavior to prevent project list scroll when pointer is on filter toolbar

Anahisv23 committed:
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593
Merge branch 'gh-pages' of https://github.com/hackforla/website into projects-and-homepages-4593

Anahisv23 commented:
> @Anahisv23 As noticed earlier, the project list scrolls when the pointer is still in the filter section on my local machine. If other reviewers do not notice that behavior, I can go ahead and approve this PR. I will wait for the other reviewer's comments. Thank you and appreciate your patience.

Hi @Thinking-Panda I just noticed this problem on my end too. Thank you for bringing it up!  I did some research and applied the overscroll-behavior css property to the filter-toolbar class. I tested it on my end and it no longer causes the project list to scroll when hovered over the filter toolbar element. Let me know how it is on your end.

Thinking-Panda reviewed (approved):
@Anahisv23 Excellent work. All scrolling works fine on my end. Thank you for your patience and perseverance.

Anahisv23 commented:
@Thinking-Panda Thank you and thank you so much for your help throughout this ticket! I'm glad everything looks good on your end. I believe now we just need @jphamtv and @mSharifHub to review this and approve it to get this merged.

jphamtv reviewed (approved):
Hi @Anahisv23 and @Thinking-Panda - I tested the scrolling behavior @Thinking-Panda mentioned and it works as expected on my end as well.

Anahisv23 commented:
Hi @jphamtv. Awesome, I'm glad its working as expected! Now we just need @mSharifHub to take a look and approve.

t-will-gillis commented:
Merging this PR b/c it has three approvals
